Cabal/Cabal.cabal

```diff
@@ -315,6 +315,7 @@ library
     Distribution.Compat.Exception
     Distribution.Compat.Graph
     Distribution.Compat.Internal.TempFile
+    Distribution.Compat.Prelude
```

Comment:
Yes, I think that's fine, assuming that the module reexport drops the warning (I can't imagine it doesn't?!) Do we have a plan? (If this works out, we could probably do this to other modules in Cabal too!)